src/tools/lifter/selftest_arm.sml

```diff
@@ -925,11 +926,7 @@ in () end;
 val arm8_expected_failed_hexcodes:string list =
 [
    "9BC37C41" (* umulh x1, x2, x3 lifting of ``Imm64 ((127 >< 64) (w2w (ms.REG 3w) * w2w (ms.REG 2w)))`` failed *),
-   "9B437C41" (* smulh x1, x2, x3 lifting of ``Imm64 ((127 >< 64) (sw2sw (ms.REG 3w) * sw2sw (ms.REG 2w)))`` failed *),
```

Comment:
I don't understand why this happened? Does something work that didn't work before?

Reply:
Four of the hexcodes that were listed as "expected to fail" actually worked. I don't think this was fixed by k14, since they were previously listed as "fixed" in the log. I removed them from the list of hexcodes expected to fail, which explains the changes.

Reply:
great. nice!